testsetups/profile.go

```diff
@@ -17,9 +18,8 @@ type ChainlinkProfileTest struct {
 	Inputs       ChainlinkProfileTestInputs
 	TestReporter testreporters.ChainlinkProfileTestReporter
 
-	env            *environment.Environment
-	networks       *blockchain.Networks
-	defaultNetwork blockchain.EVMClient
+	env *environment.Environment
+	c   blockchain.EVMClient
```

Comment:
Collapses with `client` because we have the same package, I'm personally okay if we'll have `c` `e` and `cl` because we'll always have them, or wrap them together soon.